doc/texture_packs.txt

```diff
@@ -102,6 +104,9 @@ by texture packs. All existing fallback textures can be found in the directory
 
 * `wieldhand.png`: texture of the wieldhand
 
+Note: The default textures of `player.png`, `player_back.png` and `wieldhand.png`
+are intentionally ugly. You are supposed to replace these in the game.
```

Comment:
Please remove the `are intentionally ugly` part, unnecessary and these are classic MT textures.